ml/curatedBreastData-InfoGAN/xstudy_test.py

```diff
@@ -60,8 +60,10 @@ def apply_logistic_regression(train_x, train_y, test_x, test_y):
     test_predictions = logreg.predict(test_x)
     all_train_logits = logreg.predict_proba(train_x)
     all_test_logits = logreg.predict_proba(test_x)
-    train_logits = all_train_logits[range(train_y.size), train_y.astype(np.int)]
-    test_logits = all_test_logits[range(test_y.size), test_y.astype(np.int)]
+    #train_logits = all_train_logits[range(train_y.size), train_y.astype(np.int)]
+    #test_logits = all_test_logits[range(test_y.size), test_y.astype(np.int)]
+    train_logits = all_train_logits[:, 1]
```

Comment:
Good, but what about xgboost evalation?

Reply:
there is bug in the code below:
test_auc = roc_auc_score(test_predictions, test_logits),

should be

test_auc = roc_auc_score(test_y, test_logits)